widget/group.go

```diff
@@ -64,7 +64,7 @@ func NewGroup(title string, children ...fyne.CanvasObject) *Group {
 // This group will scroll when the available space is less than needed to display the items it contains.
 func NewGroupWithScroller(title string, children ...fyne.CanvasObject) *Group {
 	box := NewVBox(children...)
-	group := &Group{BaseWidget{}, title, box, NewScrollContainer(box)}
+	group := &Group{BaseWidget{}, title, box, NewVScrollContainer(box)}
```

Comment:
It may be excessive but I think I'd prefer leaving NewGroupWithScroller as is, but add NewGroupWithHScroller and NewGroupWithVScroller. All three of which would call "newGroupWithChild" passing a scroll container holding the content.

Reply:
Hmm, not sure:
A) how useful is a vertically scrolling horizontal list
B) surely we should be correcting the accidentally changed behaviour?

Reply:
I don't see anything about Group only being vertical. It could be a HBox in a HScroll, or a VBox in a VScroll. So maybe it is just two APIs not three: NewVGroupWithVScroll or NewHGroupWithHScroll

Reply:
All groups use a vertical layout - see https://github.com/fyne-io/fyne/blob/3e1864e1922a21b28f98f1745b09b39dfea1e1b0/widget/group.go#L56

Reply:
Sure the implementation only has vertical, but nothing in the developer docs says this is the case. I don't see a downside to giving the dev a choice here

Reply:
Ah, OK. Sure I guess, but with new APIs, my point here was that I was returning it to it's previous functionality.

Reply:
> B) surely we should be correcting the accidentally changed behaviour?

> returning it to it's previous functionality

Maybe this is the misunderstanding. I don't know what it used to do and now doesn't. Was the previous functionality only scrolling vertically?

Reply:
Yes. All scrollers were previously vertical only. Horizontal scroll was added fairly recently and this widget should have been updated